packages/help-center/src/utils.ts

```diff
@@ -25,6 +25,7 @@ const wpcomAllowedOrigins = [
 // function that tells us if we want to show the Help Center to the user, given that we're showing it to
 // only a certain percentage of users.
 export function shouldShowHelpCenterToUser( userId: number ) {
+	return true;
```

Comment:
This is just to make testing easier.

Reply:
Or it's a sneaky way to launch it to 100% users :trollface: